src/block/card.scss

```diff
@@ -0,0 +1,118 @@
+.card {
+  display: flex;
+
+  flex-direction: column;
+
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+
+  padding-left: 16px;
+  padding-right: 16px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    margin-top: 32px;
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-size: 100% 100%;
+  }
+
+  &__info {
+    margin: 0;
+    margin-top: 40px;
+    padding: 0;
+  }
+
+  &__name {
+    width: 100%;
+    height: 36px;
+    color: $main-text-color;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    text-underline-position: from-font;
+  }
+
+  &__code {
+    margin: 0;
+    margin-top: 4px;
+    padding: 0;
+
+    color: #616070;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    text-underline-position: from-font;
+    margin-bottom: 16px;
+  }
+
+  &__money {
+    display: flex;
+    width: 100%;
+    height: 18px;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price {
+    margin: 0;
+
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+    color: #616070;
+  }
+
+  &__numb {
+    margin: 0;
+
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+    color: $main-text-color;
+  }
+
+  &__button {
+    text-align: center;
+
+    margin-top: 16px;
+    background-color: $back-color-button;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    border-color: $back-color-button;
+  }
+
+  &__button-link {
+    width: 100%;
+    height: 100%;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+
+    text-underline-position: from-font;
+    text-decoration: none;
+  }
+
+  &__button:hover {
+    border: 1px solid $back-color-button;
+    background-color: $main-text-color-hover;
```

Comment:
The variable `$main-text-color-hover` is used here but it's not defined in this file. Ensure that it is defined or imported from another SCSS file.